src/MerchantCenter/MerchantStatuses.php

```diff
@@ -137,7 +139,12 @@ public function get_product_statistics( bool $force_refresh = false ): array {
 	 */
 	public function get_issues( string $type = null, int $per_page = 0, int $page = 1, bool $force_refresh = false ): array {
 		$this->maybe_refresh_status_data( $force_refresh );
-		return $this->fetch_issues( $type, $per_page, $page );
+
+		// Get only critical issues
+		$critical_issues = $this->fetch_issues( $type, $per_page, $page, self::SEVERITY_ERROR );
```

Comment:
The naming here is slightly confusing since in the Google API `error` and `critical` are different severity levels. 
It also doesn't work for critical errors since in the DB we save the severity as it's retrieved from the API.

The function `get_issue_severity` combine the severity into just two levels, but that's only returned in the responses. If we filter for errors, we need to look for severity "critical OR error" (possibly more?). Or alternatively we need to save them in the DB with just two levels.

Reply:
Hey @mikkamp 

I applied your suggestion here  https://github.com/woocommerce/google-listings-and-ads/pull/1437/commits/f04dbbbbd692ac32c3a23e9a0b3e1d574acd2ef6